libkbfs/test_util.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2017 Keybase Inc. All rights reserved.
```

Comment:
Since this isn't an exported struct, shouldn't this file end in `_test.go` so it doesn't end up in the real binary?  `interfaces_test.go` maybe?